tests/test_data/jocooks.com/jocooks.json

```
@@ -0,0 +1,88 @@
+        "1 large egg (beaten)",
+        "½ cup cornstarch",
+        "vegetable oil (for frying)"
+      ],
+      "purpose": null
+    },
+    {
+      "ingredients": [
+        "3 tablespoon butter (unsalted)",
+        "4 cloves garlic (minced)",
+        "1 tablespoon fresh ginger (minced)",
+        "¼ cup honey",
+        "¼ cup brown sugar",
+        "2 tablespoon soy sauce (low sodium)",
+        "1 tablespoon rice vinegar",
+        "1 tablespoon sesame oil",
+        "2 tablespoon gochujang"
+      ],
+      "purpose": "Korean Sauce"
+    },
+    {
+      "ingredients": [
+        "green onions",
+        "toasted sesame seeds",
+        "red chilis (sliced)"
+      ],
+      "purpose": "Garnish"
+    }
+  ],
+  "instructions": "Toss the chicken pieces through the egg first, then dredge through cornstarch.\nAdd about 3 inches of oil to a heavy bottom pan and heat to 375°F.\nAdd chicken to the pan and fry in batches about 3 to 4 minutes per batch. Transfer the chicken to a paper towel lined plate and repeat with remaining chicken.\nMelt the butter in a skillet over medium heat. Add the garlic and ginger and cook for 30 seconds or until aromatic. Stir in the brown sugar and honey and cook for about 1 minute until the brown butter dissolves. Add the soy sauce, rice vinegar, sesame oil and gochujang sauce to the skillet and stir. Cook for 30 seconds then add the chicken to the skillet and toss well with the sauce.\nServe garnished with green onions, toasted sesame seeds and red chilis.",
+  "instructions_list": [
+    "Toss the chicken pieces through the egg first, then dredge through cornstarch.",
+    "Add about 3 inches of oil to a heavy bottom pan and heat to 375°F.",
+    "Add chicken to the pan and fry in batches about 3 to 4 minutes per batch. Transfer the chicken to a paper towel lined plate and repeat with remaining chicken.",
+    "Melt the butter in a skillet over medium heat. Add the garlic and ginger and cook for 30 seconds or until aromatic. Stir in the brown sugar and honey and cook for about 1 minute until the brown butter dissolves. Add the soy sauce, rice vinegar, sesame oil and gochujang sauce to the skillet and stir. Cook for 30 seconds then add the chicken to the skillet and toss well with the sauce.",
+    "Serve garnished with green onions, toasted sesame seeds and red chilis."
+  ],
+  "language": "en-US",
+  "site_name": "Jo Cooks",
+  "title": "Korean Fried Chicken",
+  "total_time": 30,
+  "yields": "6 servings",
+  "prep_time": 15,
+  "cook_time": 15,
+  "cuisine": "Korean",
+  "nutrients": {
+    "servingSize": "1 serving",
+    "calories": "466 kcal",
+    "carbohydrateContent": "33 g",
+    "proteinContent": "34 g",
+    "fatContent": "22 g",
+    "saturatedFatContent": "13 g",
+    "cholesterolContent": "143 mg",
+    "sodiumContent": "372 mg",
+    "fiberContent": "1 g",
+    "sugarContent": "21 g"
+  },
+  "ratings": 4.63,
```

```suggestion
  },
  "equipment": [
    "12-inch Cast Iron Skillet",
    "Glass Mixing Bowl Set (3 piece)"
  ],
  "ratings": 4.63,
```

(adding the test data)